BWWalkthrough/BWWalkthroughViewController.swift

```diff
@@ -287,7 +287,7 @@ extension UIView {
         Removes only constraints pertaining to the width or height of this view.
      */
     func removeHeightWidthConstraints() {
-        for constraint in constraints() as! [NSLayoutConstraint] {
+        for constraint in constraints {
```

Comment:
Exactly that, Apple have made `constraints` a property on `UIViewController` where it was once a method, and they used the Objective-C array annotations to declare it as an `NSArray` of `NSLayoutConstraint`s.